include/wx/gtk/webview_webkit.h

```diff
@@ -114,14 +114,17 @@ class WXDLLIMPEXP_WEBVIEW wxWebViewWebKit : public wxWebView
     virtual wxString GetSelectedSource() const wxOVERRIDE;
     virtual void ClearSelection() wxOVERRIDE;
 
-    virtual void RunScript(const wxString& javascript) wxOVERRIDE;
+    virtual bool RunScript(const wxString& javascript, wxString* output = NULL) wxOVERRIDE;
 
     //Virtual Filesystem Support
     virtual void RegisterHandler(wxSharedPtr<wxWebViewHandler> handler) wxOVERRIDE;
     virtual wxVector<wxSharedPtr<wxWebViewHandler> > GetHandlers() { return m_handlerList; }
 
     virtual void* GetNativeBackend() const wxOVERRIDE { return m_web_view; }
 
+    virtual wxWebViewIEEmulationLevel GetEmulationLevel() { return wxWEBVIEWIE_EMULATION_LEVEL_EMPTY; };
+    virtual bool SetEmulationLevel(wxWebViewIEEmulationLevel level) { return false; };
```

Comment:
Will you remove this from GTK and OSX or just do what you said? @sjlamerton said something different above.